package.json

```diff
@@ -29,33 +28,32 @@
     "url": "git@github.com:webpack-contrib/cache-loader.git"
   },
   "dependencies": {
-    "async": "^2.3.0",
+    "async": "^2.4.1",
     "loader-utils": "^1.1.0",
     "mkdirp": "^0.5.1"
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-core": "^6.24.1",
-    "babel-jest": "^19.0.0",
-    "babel-loader": "^6.4.1",
+    "babel-jest": "^20.0.3",
+    "babel-loader": "^7.0.0",
```

Comment:
If you are going to update this here, open a pull request in defaults so we can get them all running on 7.x